project-incubation-exit.md

```diff
@@ -106,15 +106,15 @@ decision process by informing its members of key aspects of the project.
 
     -   Participate in Hyperledger Foundation wide security discussions.
 
--   CII Badge
+-   OpenSSF Best Practices Badge
```

Comment:
Should the current graduated projects be informed about this change?
This will be applicable if the scoring criteria for badges have changed. To let projects read through the questions on best practices and aim towards silver or gold standards.